core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -28,6 +28,7 @@ class InstallGenerator < Rails::Generators::Base
                  enum: PAYMENT_METHODS.keys,
                  default: PAYMENT_METHODS.keys.first,
                  desc: "Indicates which payment method to install."
+    class_option :with_frontend, type: :string, default: 'spree/frontend'
```

Comment:
The default value "spree/frontend" doesn't do anything. It's primarily there to be consistent with the current behavior of the install generator, which installs `solidus_frontend` (`Spree::Frontend`).